apps/docs/src/content/docs/explainers/task-modifiers.mdx

```diff
@@ -3,4 +3,158 @@ title: Task modifiers
+			uploadFile.perform(file);
+		}
+	}}>start upload</button
+>
+```
+
+You can play around with it in our Timeline and you may notice a new parameter available for you:
+initially set at 1 you can update the `max` parameter to allow for more task to run at the same
+time.
+
+<Timeline selected_task_type="enqueue" client:load />
+
+## Drop
+
+The `drop` task will simply ignore (and thus drop) every `perform` that is executed when the `max`
+amount of concurrent tasks is reached. In our example with a max concurrency of 1 for two seconds
+after the first perform no other task will be accepted.
+
+You can think of this kind of tasks as a sort of throttle and since the function will be just tossed
+away you should use this modifier only when you don't care about the "execution loss". A good
+example could be sending your mouse position to a websocket to be broadcasted to all the other
+users. You don't want to update it too often to avoid clogging your server and you don't care about
+the middle positions being lost.
+
+```svelte
+<script lamng="ts">
+	import { task, timeout } from '@sheepdogs/svelte';
+	import ws from './websocket';
+
+	const updatePosition = task.drop(async (coords: { x: number; y: number }) => {
+		await timeout(200);
+		ws.send('coords', coords);
+	});
+</script>
+
+<svelte:window
+	on:mousemove={(e) => {
+		updatePosition.perform({
+			x: e.clientX,
+			y: e.clientY,
+		});
+	}}
+/>
+```
+
+I know you are waiting for it so here's your `drop` Timeline playground:
+
+<Timeline selected_task_type="drop" client:load />
+
+## Restart
+
+The `restart` task will cancel the oldest task instance once the `max` is reached. With `max` of one
+this basically means that every time you perform again you will cancel the last task.
+
+You can think of this kind of tasks as a the good 'ol debounce: if you await for, let's say, 200ms
+before doing any action in a `restart` task you can call `perform` repeatedly and the actual task
+will be executed only once the `perform` as not been called for 200ms. This is very useful to
+perform fetch requests while the user is typing without inundating your server of requests. Once the
+user stop typing the last request will not be canceled and the request will go through.
+
+```svelte
+<script lamng="ts">
+	import { task, timeout } from '@sheepdogs/svelte';
+
+	const search = task.restart(async (query: string) => {
+		await timeout(200);
+		const res = await fetch(`/api/search?q=${query}`);
+	});
+</script>
+
+<input
+	on:input={(e) => {
+		search.perform(e.target.value);
+	}}
+/>
+```
+
+And as usual here's your `restart` Timeline playground:
+
+<Timeline selected_task_type="restart" client:load />
+
+## Keep Latest
+
+Finally `keepLatest` (we kept this latest _ba-dun-tss_) it's a mix of `drop` with a sprinkle of
+`enqueue`: once the max concurrency is reached it will drop every new `perform` but it will also
+keep the very last in queue to execute it when the max concurrency drop below the max.
```

Comment:
```suggestion
Finally, we arrive at `keepLatest` (we kept this latest _ba-dun-tss_). It's a mix of `drop` with a sprinkle of
`enqueue`: once the max concurrency is reached it will drop every new `perform` but it will also
keep the very last in the queue to execute when the max concurrency drop below the `max`.
```

"we kept this latest _ba-dun-tss_" is a nice pun but doesn't really make sense

Reply:
> "we kept this latest ba-dun-tss" is a nice pun but doesn't really make sense

Ok ok i will remove it 😦